README.md

```diff
@@ -33,26 +32,25 @@ There are several steps in the compilation proccess.
 
 # How to run
 
-You need Linux.
-So I would recommend you to use Docker.
+You need Linux, so I would recommend that you use Docker.
 
-```
+```sh
```

Comment:
hmm, the `sh` mode looks to treat the # line as a comment ?
That's not what I expect... 🤔 
<img width="290" alt="Screen Shot 2019-06-02 at 22 55 21" src="https://user-images.githubusercontent.com/188741/58762290-9054c880-8589-11e9-90fc-6d0cfe90bb4d.png">

Reply:
Maybe I'll change something later.